google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/models/FiltersTest.java

```diff
@@ -350,6 +350,21 @@ public void timestampRange() {
     assertThat(actualFilter).isEqualTo(expectedFilter);
   }
 
+  @Test
+  public void timestampAtExactTime() {
+    RowFilter actualFilter = FILTERS.timestamp().exact(20_000L).toProto();
+
+    RowFilter expectedFilter =
+        RowFilter.newBuilder()
+            .setTimestampRangeFilter(
+                TimestampRange.newBuilder()
+                    .setStartTimestampMicros(20_000L)
+                    .setEndTimestampMicros(20_000L + 1))
```

Comment:
why `+ 1` here?

Reply:
`TimestampRange#endTimestampMicros` denotes upper bound for timestamp range. If we pass the same value then no rows will return.

Reply:
The end range cap is exclusive, so for a single number, the range is [x, x+1)